deno_webgpu/command_encoder.rs

```diff
@@ -407,23 +416,33 @@ impl GPUCommandEncoder {
     fn finish(
         &self,
         #[webidl] descriptor: crate::command_buffer::GPUCommandBufferDescriptor,
-    ) -> GPUCommandBuffer {
+    ) -> Result<GPUCommandBuffer, JsErrorBox> {
         let wgpu_descriptor = wgpu_types::CommandBufferDescriptor {
             label: crate::transform_label(descriptor.label.clone()),
         };
 
+        // TODO: This is not right, it should be a validation error, and it
```

Comment:
This should be an issue but I'll hold off on making it in case there's an objection to leaving things in this state.

- [ ] Create issue

Reply:
https://github.com/gfx-rs/wgpu/issues/7812